models/models.js

```diff
@@ -17,11 +19,20 @@ exports.readAllApis = () => {
     return fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8')
         .then((data) => {
             const parsedData = JSON.parse(data);
-            console.log(parsedData)
             return parsedData;
         });
 };
 
+exports.commentsByArticle = (article_id) => {
+    return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`, [article_id])
+        .then(({ rows }) => {
+            // if (!rows.length) {
+            //     return Promise.reject({ status: 404, msg: 'not found' });
+            // }
```

Comment:
This commented out bit can be deleted now, since you're handling this in the checkExists function